tests/test_dayview.py

```diff
@@ -0,0 +1,91 @@
+from datetime import datetime, timedelta
+
+from bs4 import BeautifulSoup
+import pytest
+
+from app.database.models import Event, User
+from app.routers.dayview import DivAttributes
+
+
+@pytest.fixture
+def user():
```

Comment:
I think there is already a user fixture that you can use in more general file (I don't remember if it was merged :smile: )

Reply:
יש יוזר אחד אבל הוא רנדומלי וכרגע אני צריך אחד עם שם מוגדר בשביל הבדיקות

Reply:
So maybe change the random user to be with a specific name?
Sounds like a reasonable use-case.

Reply:
אחרי שיהיה מימוש של התחברות משתמש זה דווקא יהיה טוב לטסטים בעיניים שם משתמש רנדומלי.
כשיהיה כזה מימוש גם ככה אצתרך לחזור לעבוד על הפונקציה כדי להתאים אותה ולכן כרגע אני מעדיף להשאיר יוזר ספציפי לטסטים של הdayview...

Reply:
So maybe add a TODO? :)